src/Behat/Gherkin/Loader/CucumberNDJsonAstLoader.php

```diff
@@ -118,7 +118,7 @@ private static function getBackground(array $json)
             array_map(
                 static function ($child) {
                     return new BackgroundNode(
-                        $child['background']['name'],
+                        isset($child['background']['name']) ? $child['background']['name'] : null,
```

Comment:
why not using `??`

Reply:
I just forgot what versions we support

Reply:
Well, you used `??` in the same diff a few lines below :smile: